src/lib.rs

```diff
@@ -83,14 +83,14 @@ pub fn fmt(tree: &[usize]) -> String {
     if !list[i] {
       continue;
     }
-    let depth = flat_tree::depth(i as u64);
+    let depth = flat_tree::depth(i as usize);
     let val = format!("{:width$}", i, width = width);
     matrix[i as usize][depth as usize] = val;
 
-    if let Some(children) = flat_tree::children(i as u64) {
-      add_path(&list, &mut matrix, children.0, i as u64, depth, 1);
+    if let Some(children) = flat_tree::children(i as usize) {
+      add_path(&list, &mut matrix, children.0, i as usize, depth, 1);
       if (children.1 as usize) < list.len() {
-        add_path(&list, &mut matrix, children.1, i as u64, depth, -1);
+        add_path(&list, &mut matrix, children.1, i as usize, depth, -1);
```

Comment:
We can remove the `as usize` because it's already `usize`! :D Type inference ftw!